crates/store/src/db/models/queries.rs

```diff
@@ -1072,7 +1071,152 @@ pub(crate) fn unconsumed_network_notes(
         ),
     )
     .filter(schema::notes::execution_mode.eq(0_i32))
-    .filter(schema::notes::consumed.eq(consumed_to_raw_sql(false)))
+    .filter(schema::notes::consumed_block_num.eq(None::<i64>))
```

Comment:
@drahnr I'm failing to retrieve any notes after changing `consumed` to `consumed_block_num` with the diesel setup. It might be to do with `None`, I know you had a warning about `None<T>` somewhere.

Test `sql_unconsumed_network_notes` fails on this branch. LMK if you have any ideas around that schema change.

Reply:
fix was
```rust
.filter(schema::notes::consumed_block_num.is_null())
```